config.json

```diff
@@ -295,6 +295,37 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "icon": "features-cross-platform",
+      "title": "Cross-platform",
+      "content": "Cross-compile to 14 targets, including Javascript, PHP7, C++, Lua, the JVM, and the HashLink VM."
+    },
+    {
+      "icon": "features-multi-paradigm",
+      "title": "Multi-paradigm",
+      "content": "Haxe supports OOP, metaprogramming, and many functional features."
+    },
+    {
+      "icon": "features-type-system",
+      "title": "Advanced type system",
+      "content": "A robust and sohpisticated type system with type inference, generics, abstracts, and much more."
+    },
+    {
+      "icon": "features-general-purpose",
+      "title": "General purpose",
+      "content": "Haxe excels in game development but is used in many domains, including web, mobile, and desktop."
+    },
+    {
+      "icon": "features-native",
+      "title": "Native access",
+      "content": "Haxe externs enable type-safe interaction with the underlying target, providing access to native APIs."
```

Comment:
This field exceeds the maximum length of 100 by just a bit. See [the docs](https://github.com/exercism/docs/blob/main/anatomy/tracks/config-json.md#key-features).

Reply:
Ah, last minute wording change. I missed that :-)